File: system-test/dotnet/SystemTests/Legacy/Mod06/SmokeTests/External/ErpSmokeTest.cs
using Common;
using SystemTests.Legacy.Mod06.Base;
using Shouldly;
using Xunit;

namespace SystemTests.Legacy.Mod06.SmokeTests.External;

public class ErpSmokeTest : BaseChannelDriverTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToErp()
    {
        var result = await _erpDriver!.GoToErpAsync();
        result.ShouldBeSuccess();
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod06/smoke/external/ErpSmokeTest.java
package com.mycompany.myshop.systemtest.legacy.mod06.smoke.external;

import com.mycompany.myshop.systemtest.legacy.mod06.base.BaseChannelDriverTest;
import org.junit.jupiter.api.Test;

import static com.mycompany.myshop.testkit.common.ResultAssert.assertThatResult;

class ErpSmokeTest extends BaseChannelDriverTest {
    @Test
    void shouldBeAbleToGoToErp() {
        var result = erpDriver.goToErp();
        assertThatResult(result).isSuccess();
    }
}


